suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -52,23 +68,34 @@ export const useAddCoinAccount = () => {
 
     const availableNetworkAccountTypes = useMemo(() => {
         // first account type for every network is set to normal and represents default type
-        const availableTypes: Map<NetworkSymbol | undefined, AccountType[]> = new Map();
-        Object.keys(networks).forEach(symbol =>
+        const availableTypes: Map<NetworkSymbol | undefined, [AccountType, ...AccountType[]]> =
```

Comment:
Why is there an option to have undefined as key? 😄 Seems hacky, maybe I'm wrong...

```suggestion
        const availableTypes: Map<NetworkSymbol, [AccountType, ...AccountType[]]> =
```

Reply:
5c5f2af2c4